content/en/blog/runtime-security.md

```diff
@@ -0,0 +1,276 @@
+
+As we begin to layer software it is critical to understand which layers these implementations are running at, or more importantly if the access control implementation is found at the same layer it is trying to protect. Furthermore we need to understand any interprocess communication that will be required for an access control system to work. 
+
+An easy to understand example of when prevention tooling is running at the same layer it is trying to protect is [Kubernetes Admission Controllers](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#what-are-they) attempting to enforce policy. The Kubernetes documentation suggests a client server model for parsing and enforcing arbitrary policy, and even has documentation on [how to build your own](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#write-an-admission-webhook-server).
+
+#### Policy
+
+A well known implementation of using Kubernetes Admissions Controllers is [Gatekeeper](https://github.com/open-policy-agent/gatekeeper) which is part of the [Open Policy Agent (OPA)](https://www.openpolicyagent.org/docs/latest/#overview). The [documentation advertises policy decoupling](https://www.openpolicyagent.org/docs/latest/philosophy/#policy-decoupling). Which is a deliberate attempt to seperate the policy enforcement logic, from the gateway itself. An engineer could presumably use a tool like Kubernetes to manage policy enforcement logic in the same cluster it would be attempting to protect.
+
+Furthermore there are resources in Kubernetes specifically designed to prevent exactly what this paper is exploiting. These are called [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#policy-reference) or PSPs for short. These are a prevention tactic to prevent various types of Pods from being deployed.
+
+### Detection 
+
+In this proposal we break down detection into two subsets **observability** and **alerting**.
+
+It is important to understand that both **observability** and **alerting** are commonly used to solve many problems in Kubernetes. The term **detection** applies when a user applies both of them to solve a security concern.
+
+#### Observability
+
+*Note*: in this document we use the term `observability` defined [here](https://en.wikipedia.org/wiki/Observability):
+
+> In control theory, observability is a measure of how well internal states of a system can be inferred from knowledge of its external outputs.
+
+The practice of observing events in computer science is also not a new concept. Everything from a simple logfile, to advanced systems observability could in theory be used for observability purposes. 
```

Comment:
A comma after `observability` and `could in theory` should be `could, in theory`